docs/content/configuration/creating_pvc.md

```diff
@@ -33,6 +35,8 @@ Create a PVC YAML file, similar to the following `demo-pvc-file-system.yaml` fil
     apiVersion: v1
     metadata:
       name: demo-pvc-file-system
+      labels:
+        volumegroup: demo-volumegroup
```

Comment:
can we add it just once for PVC for vg?

Reply:
like [here](https://github.com/IBM/ibm-block-csi-driver/blob/018a248f09c2036ea55ed4be7068bd12df21cd0e/deploy/kubernetes/examples/demo-pvc-in-volume-group.yaml)

Reply:
added an example for VG
"Creating a PVC with `volume_group` defined in StorageClass" in this file

Reply:
reviewed on webex